pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.pkgproj

```diff
@@ -70,13 +70,13 @@
       <Version>4.0.1</Version>
     </NETCoreApp10Dependency>
     <NETCoreApp10Dependency Include="Microsoft.NETCore.DotNetHostPolicy">
-      <Version>1.0.8</Version>
+      <Version>1.0.10</Version>
```

Comment:
@safern is there a reason this wasn't updated to 1.0.9 last time?

Reply:
I might have missed it.